tribute/tribute.html

```diff
@@ -4,10 +4,24 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link type="text/css" rel="stylesheet" href="tribute.css"/>
     <title>Document</title>
 </head>
 <body>
-    <p>tribute</p>
+    <main id="main">
```

Comment:
remove the ids.
Those are only needed if you want to do something special.
https://css-tricks.com/the-difference-between-id-and-class/

Reply:
I know ids have highest pref but this was part of the project.
if I create classes it would not let me pass the tests which are at 
https://codepen.io/freeCodeCamp/pen/MJjpwO

Reply:
ok, but you can remove the styling from ids to classes.